pkgs/applications/video/mpv/default.nix

```diff
@@ -129,7 +129,7 @@ in stdenv.mkDerivation rec {
   ++ lib.optionals waylandSupport [ wayland-scanner ];
 
   buildInputs = [
-    ffmpeg
+    ffmpeg_5
```

Comment:
Considering the code compiles with both, I would much prefer that this override is done in `all-packages.nix` instead of here. The reason is that if someone wants to change the `ffmpeg` back to 4 for some, it makes much more sense to do:

```
mpv = mpv.override { ffmpeg = ffmpeg_4; }
```

Then:

```
mpv = mpv.override { ffmpeg_5 = ffmpeg_4; }
```

Reply:
Because of some dark magic about splicing, this is being discouraged.

Reply:
> Because of some dark magic about splicing, this is being discouraged.

Would like more context about this.

Reply:
https://github.com/NixOS/nixpkgs/issues/204303

I see.

Reply:
I forgot the exact number of the issue at that moment. Apologies.